code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/request/handler/GenericWhitelistedProxy.java

```diff
@@ -174,7 +174,7 @@ protected boolean isAllowed(HttpServletRequest servletRequest, String host) {
     @SuppressWarnings("unchecked")
     @Override
     protected void generate(final ApiRequest request) throws IOException {
-        if (request.getVersion() == null || !ALLOW_PROXY.get())
+        if (!ALLOW_PROXY.get())
```

Comment:
Assuming the above change to remove `apiRequest.setVersion(null);` is cool, request.getVersion() will never not be set.